models.py

```diff
@@ -67,7 +69,6 @@ class Jurisdiction(db.Model):
 class User(db.Model):
     id = db.Column(db.String(200), primary_key=True)
     email = db.Column(db.String(200), unique=True, nullable=False)
-    external_id = db.Column(db.String(200), unique=True, nullable=False)
```

Comment:
I think I like this removal, but I wonder if it might be useful to eventually store the auth0 user id.

Reply:
Yes. But I didn't have an obvious reason to have it, and I wasn't sure what to set it to. I could make it nullable instead?

Reply:
Yes good idea, make it nullable.

Reply:
Done.